services/blockchain-indexer/shared/dataService/business/interoperability/blockchainApps.js

```diff
@@ -71,11 +74,30 @@ const getBlockchainApps = async (params) => {
 
 	const total = await blockchainAppsTable.count(params);
 
-	blockchainAppsInfo.data = await blockchainAppsTable.find(
+	const resultSet = await blockchainAppsTable.find(
 		{ ...params, limit: params.limit || total },
 		Object.getOwnPropertyNames(blockchainAppsTableSchema.schema),
 	);
 
+	const { chainID } = (await getNetworkStatus()).data;
+
+	blockchainAppsInfo.data = await BluebirdPromise.map(
+		resultSet,
+		async blockchainAppInfo => {
+			const { escrowedAmounts } = await requestConnector('getEscrowedAmounts');
+			const escrow = escrowedAmounts.filter(e => e.escrowChainID === blockchainAppInfo.chainID);
+
+			return {
+				...blockchainAppInfo,
+				escrow: escrow.length ? escrow : [{
+					tokenID: chainID.substring(0, 2).padEnd(16, '0'),
```

Comment:
`2` in `substring` and `16` in `padEnd` can be set as constants.
```suggestion
					tokenID: chainID.substring(0, NETWORK_ID_LENGTH * 2).padEnd(CHAIN_ID_LENGTH * 2, '0'),
```